plugins/mcreator-localization/help/default/entity/bounding_box.md

```diff
@@ -1,5 +1,6 @@
  This parameter controls the size of your mob's hitbox in blocks, as well as the size of its shadow. 
  The first option sets the width and depth, whilst the second option sets the height. 
- The final option sets size in blocks for the radius of the mob's shadow.
+ The third option sets size in blocks for the radius of the mob's shadow.
+ The final option sets mounted entity Y offset.

```

Comment:
Please explain a bit more what "Y offset" does, or this addition does not provide any additional value.

Reply:
It is used. But if you change tip, the tip should include how it works.